.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "build"]
 	path = build
-	url = https://github.com/upbound/build
+	url = https://github.com/hasheddan/build
+	branch = go1.14.3
```

Comment:
Should this be `go1.14.4`?

Reply:
@muvaf this is just the branch name, which I had confusingly named `go1.14.3` even though it is bumping to `go 1.14.4`. You can see the current status of this branch here: https://github.com/upbound/build/pull/98